content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,225 @@ section: doc
 At the end of the tutorial you will have a cluster with Jenkins up and running.
 
-1. We will provision a new Kubernetes Cluster for you if, you already have one skip to <<Step 2 deploy IBM Cloud Block Storage plug-in,step **2**>>
-2. We will deploy  the <<Step 2 deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if already have it skip to <<Step 3 deploy Jenkins,step **3**>>
-3. <<Step 3 deploy Jenkins,Jenkins deployment>>
+. We will provision a new Kubernetes cluster for you, if you already have one skip to <<Deploy IBM Cloud Block Storage plug-in,step 2>>.
+. We will deploy  the <<Deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if you already have it skip to <<Deploy Jenkins,step 3>>.
+. We will <<Deploy Jenkins,deploy Jenkins>>.
+. We will <<Verify Jenkins installation,verify jenkins installation>>.
+
+== Provision Kubernetes cluster
+
+. Select *Catalog* in the upper left corner.
+. Select *Service* from the catalog.
+. Search for *Kubernetes Service* and select it.
++
+image::tutorials/IBMCloud/kubernetes-select.png[IBM Kubernetes selection]
+
+You are now at the Kubernetes deployment page, and you need to specify some details about the cluster.
+
+. Choose a plan type of *standard* or *free*.
+The free plan only has one worker node and no subnet.
+To provision a standard cluster, you will need to upgrade your account to Pay-As-You-Go.
+.. To upgrade to a Pay-As-You-Go account, complete the following steps:
+... In the console, go to *Manage* > *Account*.
+... Select *Account settings*, and then select *Add credit card*.
+... Enter your payment information, select *Next*, and then submit your information.
+. Choose *Classic* or *VPC*, read the link:https://cloud.ibm.com/docs/containers?topic=containers-infrastructure_providers[docs], and then choose the most suitable type for yourself.
++
+image::tutorials/IBMCloud/infra-select.png[IBM infrastructure selection]
+
+. Now choose your location settings.
+For more information please visit link:https://cloud.ibm.com/docs/containers?topic=containers-regions-and-zones#zones[Locations].
+... Choose *Geography* (continent).
++
+image::tutorials/IBMCloud/location-geo.png[IBM geo location]
+
+... Choose *Single* or *Multizone*.
+In single zone, your data is kept in only one datacenter.
+With Multizone, it is distributed to multiple datacenters, and thus safer in an unforeseen zone failure.
++
+image::tutorials/IBMCloud/location-avail.png[IBM available locations]
+
+... Choose a *Worker Zone* if using Single zones or **Metro** if using Multizone.
++
+image::tutorials/IBMCloud/location-worker.png[IBM worker location]
+
+* If you wish to use Multizone please set up your account with link:https://cloud.ibm.com/docs/dl?topic=dl-overview-of-virtual-routing-and-forwarding-vrf-on-ibm-cloud[VRF] or link:https://cloud.ibm.com/docs/vlans?topic=vlans-vlan-spanning#vlan-spanning[enable Vlan spanning].
```

Comment:
If this should be a sub-step under the Locations step, I think we can modify the list type slightly.
```suggestion
.. If you wish to use Multizone please set up your account with link:https://cloud.ibm.com/docs/dl?topic=dl-overview-of-virtual-routing-and-forwarding-vrf-on-ibm-cloud[VRF] or link:https://cloud.ibm.com/docs/vlans?topic=vlans-vlan-spanning#vlan-spanning[enable Vlan spanning].
```